server/sonar-web/src/main/js/apps/projects/store/actions.js

```diff
@@ -180,3 +180,16 @@ export const fetchMoreProjects = (query, isFavorite, organization) =>
     });
     return searchProjects(data).then(onReceiveMoreProjects(dispatch), onFail(dispatch));
   };
+
+export const setProjectTags = (project, tags) =>
+  (dispatch, getState) => {
+    const previousTags = getState().components.byKey[project].tags;
+    dispatch(receiveProjectTags(project, tags));
+    return apiSetProjectTags({ project, tags: tags.join(',') }).then(
+      () => {},
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png 'Severity: CRITICAL') Add a nested comment explaining why this function is empty or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS1186)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Either remove or fill this block of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AEmptyBlock)